app/serializers/search/postgres_comment_serializer.rb

```diff
@@ -0,0 +1,41 @@
+module Search
+  # TODO[@atsmith813]: Rename this to CommentSerializer once Elasticsearch is removed
+  class PostgresCommentSerializer < ApplicationSerializer
+    attribute :id, &:search_id
+
+    attributes :path, :public_reactions_count
+
+    attribute :body_text, &:body_markdown
+
+    attribute :class_name do |_comment, _params|
+      "Comment"
+    end
```

Comment:
This can  be a one liner:

```suggestion
    attribute :class_name, -> { "Comment" }
```